.github/fabricbot.json

```
@@ -0,0 +1,3274 @@
+      "actions": [
+        {
+          "name": "addReply",
+          "parameters": {
+            "comment": "Hello. I see that you've just added `Needs: Author Feedback` label to this PR.\nThat label is for Issues and not for PRs. Don't worry, I'm going to replace it with the correct one."
+          }
+        },
+        {
+          "name": "removeLabel",
+          "parameters": {
+            "label": "Needs: Author Feedback"
+          }
+        },
+        {
+          "name": "addLabel",
+          "parameters": {
+            "label": "pr: pending author input"
+          }
+        }
+      ]
+    }
+  },
+  {
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "PullRequestResponder",
+    "version": "1.0",
+    "config": {
+      "conditions": {
+        "operator": "and",
+        "operands": [
+          {
+            "name": "isAction",
+            "parameters": {
+              "action": "opened"
+            }
+          },
+          {
+            "name": "activitySenderHasPermissions",
+            "parameters": {
+              "permissions": "read"
+            }
+          },
+          {
+            "operator": "not",
+            "operands": [
+              {
+                "name": "isActivitySender",
+                "parameters": {
+                  "user": "dotnet-maestro"
+                }
+              }
+            ]
+          },
+          {
+            "operator": "not",
+            "operands": [
+              {
+                "name": "isActivitySender",
+                "parameters": {
+                  "user": "dotnet-maestro-bot "
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "eventType": "pull_request",
+      "eventNames": [
+        "pull_request",
+        "issues",
+        "project_card"
+      ],
+      "taskName": "Label community PRs with `community contribution` label",
+      "actions": [
+        {
+          "name": "addLabel",
+          "parameters": {
+            "label": "community-contribution"
+          }
+        }
+      ]
+    }
+  },
+  {
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "PullRequestResponder",
+    "version": "1.0",
+    "config": {
+      "conditions": {
+        "operator": "and",
+        "operands": [
+          {
+            "name": "isAction",
+            "parameters": {
+              "action": "merged"
+            }
+          },
+          {
+            "name": "prTargetsBranch",
+            "parameters": {
+              "branchName": "main"
+            }
+          }
+        ]
+      },
+      "eventType": "pull_request",
+      "eventNames": [
+        "pull_request",
+        "issues",
+        "project_card"
+      ],
+      "actions": [
+        {
+          "name": "addMilestone",
+          "parameters": {
+            "milestoneName": "7.0-preview2"
```

Can we make this
```suggestion
            "milestoneName": "7.0-preview3"
```
and merge after I branch tonight❔ Or should we get this in quick then follow up almost immediately w/ a 'main'-only PR to make this change❔

---

I don't think I'm looking at the latest version but also see no indication of a new commit taking this change. I keep making the suggestion and then am unable to commit it.